test/Microsoft.Sbom.Api.Tests/Config/ConfigSanitizerTests.cs

```diff
@@ -243,7 +242,40 @@ public void UserProviderNamespaceUriBaseShouldReturnDefaultValue_Succeeds()
 
             mockAssemblyConfig.VerifyGet(a => a.DefaultSBOMNamespaceBaseUri);
             mockAssemblyConfig.VerifyGet(a => a.DefaultSBOMNamespaceBaseUriWarningMessage);
-            mockAssemblyConfig.VerifyNoOtherCalls();
+        }
+
+        [TestMethod]
+        public void ShouldGetPackageSupplierFromAsseblyConfig_Succeeds()
+        {
+            var organization = "Contoso International";
+            mockAssemblyConfig.SetupGet(a => a.DefaultPackageSupplier).Returns(organization);
+            var config = GetConfigurationBaseObject();
+
+            config.ManifestToolAction = ManifestToolActions.Validate;
+            configSanitizer.SanitizeConfig(config);
+
+            Assert.AreEqual(organization, config.PackageSupplier.Value);
+
+            mockAssemblyConfig.VerifyGet(a => a.DefaultPackageSupplier);
+        }
+
+        [TestMethod]
+        public void ShouldNotOverridePackageSupplierIfProvided_Succeeds()
+        {
+            var organization = "Contoso International";
+            var actualOrg = "Contoso";
+            mockAssemblyConfig.SetupGet(a => a.DefaultPackageSupplier).Returns(organization);
+            var config = GetConfigurationBaseObject();
+            config.PackageSupplier = new ConfigurationSetting<string>
+            {
+                Source = SettingSource.CommandLine,
+                Value = actualOrg
+            };
+
+            config.ManifestToolAction = ManifestToolActions.Validate;
+            configSanitizer.SanitizeConfig(config);
+
+            Assert.AreEqual(actualOrg, config.PackageSupplier.Value);
```

Comment:
AreEqual's arguments are expected followed by actual.

Reply:
Oh this actually maybe just a misleading name for a variable, `actualOrg` is an expected value for test, `config.PackageSupplier.Value` is an actual :)